src/lib/Scenes/Fair/Components/SearchLink.tsx

```diff
@@ -30,7 +30,7 @@ export class SearchLink extends React.Component<Props> {
 
   render() {
     return (
-      <TouchableWithoutFeedback onPress={() => this.handlePress}>
+      <TouchableWithoutFeedback onPress={() => this.handlePress()}>
```

Comment:
Just noticed the search bar on Fair was broken and bundled that fix there as it's very small